modules/rosa-delete-dedicated-admins.adoc

```diff
@@ -16,7 +16,7 @@ Only the user who created the cluster can revoke access for a `dedicated-admin`
 
```

Comment:
There is no need to 'logged in to the cluster' unless users want to confirm if the user is removed from the group.

Reply:
The 2nd. step in the procedure is to verify that the user no longer is listed in the output, so I'm retaining this piece of info.